src/file_validator.py

```diff
@@ -321,11 +321,24 @@ def validate_all_files(self, submission_id):
                 if '/log' in file.key:
                     break
                 file_name = file.key.split('/')[-1]
-                
+               
                 if file_name not in manifest_file_names:
-                    msg = f'File “{file_name}”: no record found.'
+                    file_batch = self.mongo_dao.find_batch_by_file_name(submission_id, "data file", file_name)
+                    batchID = file_batch[ID] if file_batch else "-"
+                    displayID = file_batch["displayID"] if file_batch else "-"
+                    msg = f'Extra data file “{file_name}” is found in CRDC datahub storage.'
```

Comment:
According to the error message document, this should be "**File “{file_name}”: no record found**"

Reply:
Updated.